### PR TITLE
Delete <Вхід> in sidebar,add url /admin for admin

### PR DESCRIPTION
**Pull request by volodymyryaniv**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -138,6 +139,7 @@ const App = ({
     <div className="App">
       <div className={classes.mainStyle}>
         <Switch>
+        <Route path="/admin" exact component={SignInModal} />
```
Comment (klerik) [inline client/src/App.js:142]: Please check linter for indents

File: client/src/modules/Auth/submodules/SignIn/components/ButtonSignIn.js [hunk 2]
```diff
@@ -17,20 +17,17 @@ const useStyles = makeStyles({
...
-  );
   return (
-    <Link to="/account">
+    <>
+    {user && <Link to="/account">
       <Tooltip title="Особистий кабінет">
         <PersonIcon fontSize="large" className={classes.personIcon} />
       </Tooltip>
-    </Link>
+    </Link>}
```
Comment (klerik) [inline client/src/modules/Auth/submodules/SignIn/components/ButtonSignIn.js:29]: Please check linter for indents - and delete an unnecessary component that you are not using - Button

File: client/src/shared/ModalWrapper/index.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
-import React, { useState } from 'react';
+import React, { useState,useEffect} from 'react'; //
+import { useHistory,useLocation } from 'react-router-dom';  //
```
Comment (klerik) [inline client/src/shared/ModalWrapper/index.js:2]: delete an unnecessary component that you are not using

**Review by klerik — CHANGES_REQUESTED** [review]

File: client/src/modules/Auth/submodules/SignIn/components/ButtonSignIn.js (hunk 2 above)
Comment (volodymyryaniv) [reply client/src/modules/Auth/submodules/SignIn/components/ButtonSignIn.js]: Fixed

File: client/src/shared/ModalWrapper/index.js (hunk 3 above)
Comment (volodymyryaniv) [reply client/src/shared/ModalWrapper/index.js]: Done

File: client/src/modules/Auth/submodules/SignIn/components/ButtonSignIn.js [hunk 4]
```diff
@@ -17,20 +17,17 @@ const useStyles = makeStyles({
   }
 });
 
-const ButtonSignIn = ({ handleOpen, user }) => {
```
Comment (klerik) [inline client/src/modules/Auth/submodules/SignIn/components/ButtonSignIn.js:20]: handleOpen was removed but I still see this prop in propTypes

File: client/src/modules/Auth/submodules/SignIn/components/ButtonSignIn.js [hunk 5]
```diff
@@ -17,20 +17,17 @@ const useStyles = makeStyles({
...
-        <PersonIcon fontSize="large" className={classes.personIcon} />
-      </Tooltip>
-    </Link>
+    <>
+      {user && <Link to="/account">
+        <Tooltip title="Особистий кабінет">
+          <PersonIcon fontSize="large" className={classes.personIcon} />
+        </Tooltip>
+      </Link>}
+    </>
```
Comment (klerik) [inline client/src/modules/Auth/submodules/SignIn/components/ButtonSignIn.js:30]: ESLint: Missing parentheses around multilines JSX(react/jsx-wrap-multilines) - please, add parentheses

**Review by klerik — CHANGES_REQUESTED** [review]

File: client/src/modules/Auth/submodules/SignIn/components/ButtonSignIn.js (hunk 4 above)
Comment (volodymyryaniv) [reply client/src/modules/Auth/submodules/SignIn/components/ButtonSignIn.js:20]: Done

File: client/src/modules/Auth/submodules/SignIn/components/ButtonSignIn.js (hunk 5 above)
Comment (volodymyryaniv) [reply client/src/modules/Auth/submodules/SignIn/components/ButtonSignIn.js:32]: Fixed

**Pull request merged**